tests/core/operations/test_transform.py

```diff
@@ -117,65 +118,88 @@ def _unpad_rasters(sdata: SpatialData) -> SpatialData:
     return SpatialData(images=new_images, labels=new_labels)
 
 
-# TODO: when the io for 3D images and 3D labels work, add those tests
-def test_transform_image_spatial_image(images: SpatialData):
-    sdata = SpatialData(images={k: v for k, v in images.images.items() if isinstance(v, SpatialImage)})
+def _postpone_transformation(
+    sdata: SpatialData, from_coordinate_system: str, to_coordinate_system: str, transformation: BaseTransformation
+):
+    for element in sdata._gen_elements_values():
+        d = get_transformation(element, get_all=True)
+        assert isinstance(d, dict)
+        assert len(d) == 1
+        t = d[from_coordinate_system]
+        sequence = Sequence([t, transformation])
+        set_transformation(element, sequence, to_coordinate_system)
```

Comment:
Perhaps better to move this to a `_test_utils.py` in src/spatialdata.

Reply:
I would keep this function here, at least for the moment, because I think it's better for the user to do this operation manually. In this case we assume that len(d) == 1, but in more complex scenario you may have an intricate graph of transformations, and this function could compromise some alignments to other coordinate systems.